docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -32,112 +32,229 @@ For example:
+
+```yaml
+    - name: Push OctoPetShop Database
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.Database.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+    - name: Push OctoPetShop Web
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.Web.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+    - name: Push OctoPetShop ProductService
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.ProductService.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+    - name: Push OctoPetShop ShoppingCartService
+      run: |
+        octo push --package="$GITHUB_WORKSPACE/artifacts/OctoPetShop.ShoppingCartService.$PACKAGE_VERSION.zip" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
+```
 
-  build:
-    name: Build
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v2
-      
-      - name: Install Octopus CLI
-        uses: !include <image-version-install-octopus-cli-action>
-        with:
-          version: latest
-          
-      - name: list-octopusdeploy-deployments
-        run: octo list-deployments --server=${{ env.serverURL }} --apiKey=${{ secrets.apiKey }}
+### Creating a release
+Using the Octopus CLI Action, add a step to issue the [create-release](https://octopus.com/docs/octopus-rest-api/octopus-cli/create-release) command to create a release
+
+```yaml
+    - name: Create release
+      run: |
+        octo create-release --project="Octo Pet Shop" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}"
 ```
 
+### Deploying a release
+To have your GitHub Action build deploy a release, add a step with the [deploy-release](https://octopus.com/docs/octopus-rest-api/octopus-cli/deploy-release) command.
+:::hint
+Use either the `--progress` or `--waitForDeployment` switches to have the build wait for the deployment to complete and report success or failed.  `--progress` will display messages from Octopus itself whereas `--waitForDeployment` will simply wait for Octopus to report success or failure.
+
+```yaml
+    - name: Deploy release
+      run: |
+        octo deploy-release --project="Octo Pet Shop" --server="${{ secrets.OCTOPUSSERVERURL }}" --apiKey="${{ secrets.OCTOPUSSERVERAPIKEY }}" --space="${{ secrets.OCTOPUSSERVER_SPACE }}" --releaseNumber="latest" --deployTo="Development"
+```
+:::
+
```

Comment:
Is it possible to have a section at the end that combines all the YAML from the previous section so a person can see "the full script"?

Reply:
Yes, I can do that.  Would it make sense to have it be a collapsed section that the reader can expand?

Reply:
I like that idea. I find the collapsed sections sometimes less obvious, so my preference would be a sentence to state to expand it (as silly as it sounds)

Reply:
Collapsable section added with full YAML.